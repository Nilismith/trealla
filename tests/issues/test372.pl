:-initialization(main).
:-set_prolog_flag(strict_iso,off).

answer(41) :- 1 =:= 2, fail.
answer(42).

main :- H = answer(X), clause(H, Body), writeln((H :- Body)), fail.
main.
