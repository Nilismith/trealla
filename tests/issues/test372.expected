answer(41):-1=:=2,fail
answer(42):-true
